aten/src/ATen/Parallel.h

```diff
@@ -141,4 +143,13 @@ inline scalar_t parallel_reduce(
   }
 }
 
+class C10_API PTThreadPool : public c10::ThreadPool {
```

Comment:
shouldn't it be AT_API ?

it's unfortunate we need to create an extra class here because of c10/aten boundary. Karl and I are working on it :)

Reply:
I don't see AT_API used (that is I see it only in older /experimental/ paths), I think we use C10_API or TORCH_API, and in any case all of this is doing the same thing

Reply:
or, did you mean that we'd want to reintroduce AT_API? I thought it's better to agree to one thing